From 2782b95da52375c532fd019b14bf642ff84ac84d Mon Sep 17 00:00:00 2001
From: Andy Ross <andrew.j.ross@intel.com>
Date: Wed, 28 May 2014 15:54:55 -0700
Subject: [PATCH 053/117] mt9m114: Add ACPI ID

Devices with this sensor in the market are already shipping with
INT33F0.  Leave APTN1040 in place for now for compatibility, but
hopefully we can kill this extra one before it ships.

Signed-off-by: Andy Ross <andrew.j.ross@intel.com>
---
 drivers/external_drivers/camera/drivers/media/i2c/mt9m114.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/drivers/external_drivers/camera/drivers/media/i2c/mt9m114.c b/drivers/external_drivers/camera/drivers/media/i2c/mt9m114.c
index 679a115..6a0d2c1 100644
--- a/drivers/external_drivers/camera/drivers/media/i2c/mt9m114.c
+++ b/drivers/external_drivers/camera/drivers/media/i2c/mt9m114.c
@@ -1533,7 +1533,8 @@ out_free:
 }
 
 static struct acpi_device_id mt9m114_acpi_match[] = {
-	{"APTN1040"},
+	{ "APTN1040" },
+	{ "INT33F0" },
 	{},
 };
 
-- 
1.9.3

